stdlib/public/Backtracing/ArrayImageSource.swift

```diff
@@ -0,0 +1,50 @@
+//===--- ArrayImageSource.swift - An image source backed by an Array -------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+//
+// Defines ArrayImageSource, an image source that is backed by a Swift Array.
+//
+//===----------------------------------------------------------------------===//
+
+import Swift
+
+@_implementationOnly import OS.Libc
```

Comment:
I wasn't aware of this module, do we finally have a replacement for all the numerous platform-specific libc modules?

Reply:
That is relevant here because of 1c8362b, which was needed because I did that (and because the test does slightly the wrong thing — it treats the module it's testing as the main module, which causes the deserialiser to behave differently).